main.js

```diff
@@ -0,0 +1,386 @@
+let gamesPlayed = 1;
+let startTime = Date.now();
+let turnCount = 0;
+let whoseTurn = "X";
+let occupiedCells = {
+  all: [],
+  X: [],
+  O: []
+};
+let cells = [];
+let startOverButton = document.getElementById("startOver");
+startOverButton.style = "opacity: 0";
```

Comment:
Is there a reason you didn't just do this in the style sheet? Your code would be more efficient if it was there instead of here.